subsys/bluetooth/controller/ll_sw/ti/hal/cc13xx_cc26xx/debug.h

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
Is reference, vega, an oversight?

Reply:
I mostly copied the vega implementation, which mostly copied the Nordic implementation. Not 100% finished yet, so irrelevant references will be updated once it's functional.

Reply:
Covered under the task list